README.md

```diff
@@ -117,6 +117,42 @@ Feature('Some feature', () => {
 
 <img src="doc/only-v2.png" width="400" />
 
+
+### TypeScript
+
+The TypeScript definitions are bundled together with mocha-cakes-2.
+When using TypeScript run mocha with [`mocha -r ts-node/register ...`](https://github.com/TypeStrong/ts-node).
+
```

Comment:
Moght be good to add something about needing to install @types/mocha as well, since most of the utility functions are declared there.